sdk/Trace/Tracer.php

```diff
@@ -167,7 +168,10 @@ public function getSpans(): array
     {
         return $this->spans;
     }
-
+    public function getResource(): ResourceInfo
+    {
+        return clone $this->resource;
```

Comment:
same question in re: `clone`

Reply:
Trying to ensure the immutableness of resource.  The [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/resource/sdk.md) says `A Resource is an immutable representation of the entity producing telemetry.` Similar to how we handle the immutableness of `context` in this [getter](https://github.com/open-telemetry/opentelemetry-php/blob/master/sdk/Trace/Span.php#L67).

Reply:
Ahh perfect.  Thanks for the clarification!